docs/publish-module/publish-information/meta-data-information.md

```diff
@@ -0,0 +1,78 @@
+:::
+
+## iOS Meta Data Information
+
+### Localization Settings
+
+The localization dropdown allows you to select the language in which you want to present your app’s metadata on the App Store. This feature supports multiple languages, ensuring that you can target specific demographics and cater to a global audience.
+
+When you select a language, you will provide localized versions of your app's metadata, including promotional texts, descriptions, and what’s new in this version. Localization helps in reaching a wider audience by providing information in the users' native language.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3667-meta-data-information-localization.png' />
+
+---
+
+### Fields Explained
+
+#### iOS Previews and Screenshots
+
+- **Drag and Drop**: Upload up to 3 app previews and 10 screenshots.
+  - **iPhone**: Supported resolutions are 1290 x 2796px or 2796 x 1290px.
+  - **iPad**: Supported resolutions are 2048 x 2732px or 2732 x 2048px.
+
+#### App Information
+
+- **Promotional Text**: Optional text to inform App Store visitors about any current app features without requiring an updated submission.
+- **Description**: Provide a detailed description of your app, highlighting its features and functionalities.
+- **What’s New**: Describe what’s new in this version of your app, such as new features, improvements, and bug fixes.
+- **Name**: The name of your app as it will appear on the App Store.
+- **Subtitle**: A brief summary of your app, limited to 30 characters.
+- **Keywords**: Include keywords that describe your app. Separate keywords with English commas, Chinese commas, or a mix of both.
+- **Support URL**: A URL with support information for your app. This will be visible on the App Store.
+- **Marketing URL**: A URL with marketing information about your app. This will be visible on the App Store.
+
+#### App Version Information
+
+- **Version**: The version number of the app following standard software versioning conventions.
+- **Copyright**: The name of the person or entity that owns the copyright to your app.
+
+#### App Review Information
+
+- **Sign-In Information**: If your app requires a sign-in, provide credentials that will be used by the review team.
+- **Contact Information**: Provide the name, phone number, and email of the contact person for the App Review team.
+- **Notes**: Include any additional information that you want to share with the App Review team.
+
+#### App Release Information
+
+- **App Store Version Release**: Choose how you want to release your app on the App Store:
+  - Manually release this version
+  - Automatically release this version
+  - Automatically release this version after App Review, no earlier than a specified date and time
+- **Phased Release for App Store Automatic Updates**: Opt in to gradually release updates over a 7-day period to users.
+- **Reset iOS App Store Summary Rating**: Decide if you want to reset the app's rating when the new version is released.
```

Comment:
```suggestion
- **Reset iOS App Store Summary Rating**: Decide if you want to reset the app's rating when the new version is released.

:::danger
Please note, this comes with Keep Existing Rating selected by default. If you choose to reset your ratings, it will reset all past reviews and ratings of your app.
:::
```

Reply:
@erenkan you can use ss from UI